quinn-proto/src/crypto.rs

```diff
@@ -186,3 +190,24 @@ pub trait HmacKey: Send + Sized + Sync {
     /// Method for verifying a message
     fn verify(&self, data: &[u8], signature: &[u8]) -> Result<(), ()>;
 }
+
+/// A pseudo random key for HKDF
+pub trait PseudoRandomKey: Send + Sized + Sync {
+    /// Method to build pseudo random key from existing bytes
+    fn from_secret(secret: &[u8]) -> Self;
+    /// Generate key using prk and fill the key_buffer
+    fn to_bytes(&self, random_bytes: &[u8], key_buffer: &mut [u8]);
```

Comment:
```suggestion
    fn expand(&self, random_bytes: &[u8], key_buffer: &mut [u8]);
```
Let's use the RFC terminology. `to_bytes` makes me think of e.g. [`str::as_bytes`](https://doc.rust-lang.org/std/primitive.str.html#method.as_bytes), which is a lossless, reversible encoding of the parameter--definitely not what we're going for here!